src/main/java/ch/njol/skript/config/SectionNode.java

```diff
@@ -363,7 +368,9 @@ private final SectionNode load_i(final ConfigReader r) throws IOException {
 			}
 			
 		}
-		
+		if (inBlockComment.get()) {
+			Skript.error("A block comment (###) was opened on line " + blockCommentStartLine + " but never closed.");
```

Comment:
Maybe a warning would be better here?

Reply:
No because we don't really want floating comment marks being a thing, plus it's very likely it's a mistake (e.g. somebody accidentally wrote `###` somewhere and half their file doesn't load anymore).